index.css

```diff
@@ -0,0 +1,97 @@
+/* Mobile Ready */
+@media screen and (max-width: 760px) {
+
+    [class*="col-"] {
```

Comment:
I like the idea of this, but your site doesn't seem to actually behave responsively.